frontend/packages/dev-console/locales/ja/devconsole.json

```diff
@@ -20,7 +20,7 @@
   "Upload JAR file": "JAR ファイルのアップロード",
   "Upload a JAR file from your local desktop to OpenShift": "ローカルデスクトップから OpenShift に JAR ファイルをアップロードします",
   "Run command": "コマンドの実行",
-  "Optional parameter for Node.js applications to define the npm run <command>.": "Node.js アプリケーションのオプションのパラメーターで、npm run <command> を定義します。",
+  "Optional arguments for npm run.": "Node.js アプリケーションのオプションのパラメーターで、npm run <command> を定義します。",
```

Comment:
@abhinandan13jan just curious, how did these files get updated i.e ja, ko, zh etc?

Reply:
updated all keys manually, followed by a 'yarn i18n'

Reply:
I don't think you would need to do that, can you pls revert the manual edition on locales files?

Reply:
updated